### PR TITLE
Issue for writer

### PR DESCRIPTION
**Pull request by TiruDeepthi**

### CODE REVIEW

File: src/components/NormalizationDashboard.jsx [hunk 1]
```diff
@@ -14,6 +14,7 @@ function NormalizationDashboard() {
   const navigate = useNavigate();
   const [accountDetails] = useSessionStorage("accountDetails");
   const [search, setSearch] = useState("");
+  const [searchCount, setSearchCount] = useState(false);
```
Comment (RajKumar-BackCountry) [inline src/components/NormalizationDashboard.jsx:17]: what are using this for ?

File: src/components/NormalizationDashboard.jsx [hunk 2]
```diff
@@ -222,7 +219,13 @@ function NormalizationDashboard() {
   }
 
   const handleSearchChange = (e) => {
-    setSearch(e.target.value);
+    if(e.target.value !== ""){
+      const countOfEnteredSearch = e.target.value.replaceAll(" ", "").split(",")
+      setSearchCount(countOfEnteredSearch?.length >500 ? true : false)
+    if(countOfEnteredSearch?.length<=500){
```
Comment (RajKumar-BackCountry) [inline src/components/NormalizationDashboard.jsx]: indentation not proper. Is prettier fixed in your vscode ?

File: src/components/GlobalSearch.jsx [hunk 3]
```diff
@@ -10,40 +11,62 @@ function GlobalSearch({
   onClick,
   searchValue,
   handleClear,
-  img
+  img,
+  searchCount,
+  setSearchCount
 }) {
+  const handleErrorClose = () => {
+    setSearchCount(false);
```
Comment (RajKumar-BackCountry) [inline src/components/GlobalSearch.jsx]: if it a single line change, keep it inline where you invoke this.
No need for a separate function call.

File: src/components/GlobalSearch.jsx [hunk 4]
```diff
@@ -10,40 +11,62 @@ function GlobalSearch({
...
-            src={img}
-            alt="NavBarSwitchingIcon SVG"
-            data-testid="test-img"
-          />
+          {" "}
+          {searchString}{" "}
         </Button>
+      </div>
+      {searchCount && (
+        <div className="flex items-center w-full p-4 bg-[#FC555619] text-grey-20 rounded shadow mt-2 border border-[#F95455] h-[75px]">
```
Comment (RajKumar-BackCountry) [inline src/components/GlobalSearch.jsx]: why is height in px ?

File: src/components/GlobalSearch.jsx [hunk 5]
```diff
@@ -10,40 +11,62 @@ function GlobalSearch({
...
+          <div className="mr-2">
+            <img className="h-[21px] w-[23px]" src={Warning} alt="close SVG" />
+          </div>
+          <div className="text-sm font-[14px] text-[#4D4D4D]">
+            You have entered more than 500 styles. Please remove 501 styles
```
Comment (RajKumar-BackCountry) [inline src/components/GlobalSearch.jsx]: Who gave this text ?

File: src/components/GlobalSearch.jsx [hunk 6]
```diff
@@ -10,40 +11,62 @@ function GlobalSearch({
...
+          </div>
+          <div className="flex items-center ml-auto space-x-2">
+            <Button onClick={handleErrorClose}>
```
Comment (RajKumar-BackCountry) [inline src/components/GlobalSearch.jsx]: why do you need a button wrap. why not onclick event on img itself.

File: src/components/GlobalSearch.jsx (hunk 5 above)
Comment (TiruDeepthi) [reply src/components/GlobalSearch.jsx]: Arun has given

Comment (RajKumar-BackCountry) [reply src/components/GlobalSearch.jsx]: Text is not correct. Call him and get it corrected.

File: src/components/Table.jsx [hunk 7]
```diff
@@ -704,6 +695,10 @@ export default function Table({
     );
   };
 
+  const handleStyleId = (value) => {
```
Comment (RajKumar-BackCountry) [inline src/components/Table.jsx:698]: where is this being used ?

File: src/context/normalizationDashboard.jsx [hunk 8]
```diff
@@ -155,7 +158,11 @@ function DashBoardProvider({ children }) {
         updatedAtSort,
         setUpdatedAtSort,
         assigneeSort,
-        setAssigneeSort
+        setAssigneeSort,
+        selectedStyleId,
+       setSelectedStyleId,
```
Comment (RajKumar-BackCountry) [inline src/context/normalizationDashboard.jsx]: indenation is not ok.

**Pull request merged**